### PR TITLE
mm-api-ref-docs-oauth-prompt

### PR DESCRIPTION
**Pull request by mmiele**
Added ref documentation.

### CODE REVIEW

File: libraries/botbuilder-dialogs/botbuilder/dialogs/prompts/oauth_prompt.py [hunk 1]
```diff
@@ -32,39 +32,54 @@
...
+        Creates a new instance of the :class:`OAuthPrompt` class.
+
+        :param dialogId: The Id to assign to this prompt.
+        :type dialogId: str
+        :param settings: Additional authentication settings to use with this instance of the prompt.
+        :type settings: :class:`OAuthPromptSettings`
+        :param validator: Optional, a :class:`PromptValidator` that contains additional, custom validation for this prompt.
+        :type validator: :class:`PromptValidatorContext`
+
+        .. remarks::
```
Comment (emgrol) [inline libraries/botbuilder-dialogs/botbuilder/dialogs/prompts/oauth_prompt.py]:
```suggestion
        .. note::
```

File: libraries/botbuilder-dialogs/botbuilder/dialogs/prompts/oauth_prompt.py [hunk 2]
```diff
@@ -180,6 +235,17 @@ def __init__(
...
 
     async def sign_out_user(self, context: TurnContext):
+        """Signs out the user
+
+        :param context: Context for the current turn of conversation with the user.
+        :type context:  :class:`TurnContext`    
+        :return: A :class:`Task` representing the work queued to execute.
+        :rtype: :class:`Task`            
+
+       .. remarks::
```
Comment (emgrol) [inline libraries/botbuilder-dialogs/botbuilder/dialogs/prompts/oauth_prompt.py]:
```suggestion
       .. note::
```

File: libraries/botbuilder-dialogs/botbuilder/dialogs/prompts/oauth_prompt.py (hunk 1 above)
Comment (mmiele) [reply libraries/botbuilder-dialogs/botbuilder/dialogs/prompts/oauth_prompt.py]: Fixed

File: libraries/botbuilder-dialogs/botbuilder/dialogs/prompts/oauth_prompt.py (hunk 2 above)
Comment (mmiele) [reply libraries/botbuilder-dialogs/botbuilder/dialogs/prompts/oauth_prompt.py]: Fixed

**Pull request merged**